tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -536,15 +536,16 @@ public function testGetDefaultValueDeclarationSQL()
     public function testGetDefaultValueDeclarationSQLDateTime()
     {
         // timestamps on datetime types should not be quoted
-        foreach (array('datetime', 'datetimetz') as $type) {
-
+        foreach (array('datetime', 'datetimetz', 'datetime_immutable', 'datetimetz_immutable') as $type) {
             $field = array(
                 'type' => Type::getType($type),
                 'default' => $this->_platform->getCurrentTimestampSQL()
             );
 
-            $this->assertEquals(' DEFAULT ' . $this->_platform->getCurrentTimestampSQL(), $this->_platform->getDefaultValueDeclarationSQL($field));
-
+            $this->assertEquals(
```

Comment:
Please use `self::assert*` instead, and it would be better to use `assertSame()` in this case